src/models.ts

```diff
@@ -64,3 +64,11 @@ export interface RecipeListModel {
   filters: Array<FilterModel>;
   sortOptions: Array<SortOptionModel>;
 }
+export interface SearchOptions {
```

Comment:
let's not mix response models and request models.
Keep response models in this file, and move back request model "SearchOption" into recipes.api